README.md

```diff
@@ -80,3 +82,54 @@ $ npm install
 $ npm run clean
 $ npm run build
 ```
+
+## For server-side rendering via node.js
+
+Copy `xypic.js` to your npm project, and create a `mathjax.js` in the same directory containing:
+
+```js
+import { readFileSync, writeFileSync } from 'fs';
+import { argv } from 'yargs';
+
+import * as mathjax from 'mathjax-full';
+
+mathjax
+  .init({
+    options: {
+      typesetError: (_, math, err) => console.log(math.math + ': ' + err),
+    },
+    loader: {
+      paths: { mathjax: 'mathjax-full/es5', custom: '.' },
+      require: require,
+      load: ['input/tex-full', 'output/chtml', '[custom]/xypic'],
+    },
+    tex: {
+      packages: { '[+]': ['xypic'] },
+      inlineMath: [['$', '$']],
+    },
+    chtml: {
+      fontURL: 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2',
+    },
+  })
+  .then((MathJax) =>
+    argv._.forEach((r) => {
+      // Read in the HTML file
+      const html = (MathJax.startup.document = MathJax.startup.getDocument(readFileSync(r, 'utf8')));
+
+      // xypic has used the adaptor
+      const adaptor = MathJax.startup.adaptor;
+
+      // Clear the font cache
+      html.outputJax.font.clearCache();
+
+      // Typeset the document, with the render hooks that xypic has put in place
+      html.clear().rerender();
+
+      // Output the resulting HTML in-place
+      writeFileSync(r, adaptor.doctype(html.document) + adaptor.outerHTML(adaptor.root(html.document)));
+    })
+  )
+  .catch((err) => console.log(err));
+```
+
+Now run `node -r esm mathjax.js -- index.html` to replace your HTML file with a version with all the math pre-rendered.
```

Comment:
If you don't use yargs, you will want to remove the '--' (unless you use that in your code to tell when the arguments begin in `process.argv`).